### DISCUSSION TITLE
1.5.4

### DISCUSSION THREAD

**Discussion by stnguyen90** [appwrite/appwrite#7865] [Show and tell]
## Fixes

* Bump executor version to fix docker conflict error by @stnguyen90 in https://github.com/appwrite/appwrite/pull/7804
* Fix webhooks failed connection by @stnguyen90 in https://github.com/appwrite/appwrite/pull/7848
* Fix msg91 params by @abnegate in https://github.com/appwrite/appwrite/pull/7824
* Fix functions domain permissions by @stnguyen90 in https://github.com/appwrite/appwrite/pull/7853
* Fix function build command by @abnegate in https://github.com/appwrite/appwrite/pull/7813

## Miscellaneous

* Bump console to version 4.0.5 by @stnguyen90 in https://github.com/appwrite/appwrite/pull/7863

**Full Changelog**: https://github.com/appwrite/appwrite/compare/1.5.3...1.5.4

<hr /><em>This discussion was created from the release <a href='https://github.com/appwrite/appwrite/releases/tag/1.5.4'>1.5.4</a>.</em>

**Comment by SmollaL1st** [comment]
Am I the only one who can’t log into the console after the update? (Self-hosting)

**Reply by EVDOG4LIFE** [reply]
Did you change your API key when upgrading? Ensure the `_APP_OPENSSL_KEY_V1` matches the previous verion in your .env file backup.

**Reply by SmollaL1st** [reply]
The API key was not touched, but I already solved the problem, rolled back to 1.5.3 and updated again, thanks for the help.
